ibi/coreos-installer/scripts/generate_certs.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -e
+
+# Set the cluster and user names
+CLUSTER_NAME="other-test-sno.example.com"
+USER_NAME="system:admin"
+CA_NAME="admin-kubeconfig-signer"
+LOADBALANCER_SIGNER_NAME="loadbalancer-serving-signer"
+LOCALHOST_SIGNER_NAME="localhost-serving-signer"
+SERVICE_NETWORK_SIGNER_NAME="service-network-serving-signer"
+INGRESS_OPERATOR_SIGNER_NAME="ingresskey-ingress-operator"
+
+# Set the paths for certificates and kubeconfig file
+CERT_DIR="./certs"
+CONFIG_CERT_DIR="./config/certs"
+KUBECONFIG_FILE="kubeconfig"
+
+# Create a temporary OpenSSL configuration file
+config_file=$(mktemp)
```

Comment:
Add a cleanup function with an exit trap to cleanup temporary files

Something like this:
https://github.com/javipolo/redhat-pocs/blob/d2cac74a8e7a1b7cb2c93f9fd629aa8501cc0599/vdu-microshift/download_bundle.sh#L11-L15